VimeoNetworking/Sources/Models/GCS.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  GCS.swift
+//  Vimeo
+//
+//  Created by Nguyen, Van on 11/8/18.
+//  Copyright © 2018 Vimeo. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+public class GCS: VIMModelObject
+{
+    public enum Connection: String
+    {
+        case uploadAttempt = "upload_attempt"
+    }
+    
+    @objc public private(set) var startByte: NSNumber?
+    @objc public private(set) var endByte: NSNumber?
+    @objc public private(set) var uploadLink: String?
+    @objc internal private(set) var metadata: [String: Any]?
+    
+    public private(set) var connections = [Connection: VIMConnection]()
```

Comment:
Should that guy be optional?

Reply:
I think it might be better to just instantiate the `connections` dictionary up front. The reason we keep other properties optional is because `VIMObjectMapper` is written in Objective-C, which when translated to Swift can either be optional or force-unwrapped. Having the `connections` dictionary initialized eliminate unnecessary need to unwrap optional.